src/molecules/FilterBar.tsx

```diff
@@ -33,6 +33,11 @@ const Filterbar: FunctionComponent = () => {
             background={node.color.hex}
           ></FilterButton>
         ))}
+      <FilterButton
+        to="/podcasts"
+        name="Podcasts"
+        background="#999"
```

Comment:
Maybe this should get the color from theme.ts?

Reply:
Good catch, I forgot to change that! Just added this color for testing but it should ofc come from the theme file!